packages/threat-composer/src/components/generic/WindowExporter/index.tsx

```diff
@@ -69,35 +70,23 @@ const WindowExporter: FC<PropsWithChildren<{}>> = ({ children }) => {
 
   useEffect(() => {
     window.threatcomposer.getWorkspaceList = () => workspaceList;
```

Comment:
I think it might be good to change all the methods to be undefined*able and without initializing the methods. Given that not all the methods are ready at the same time. 

Consumers are required to check whether it is not undefined, like `window.threatcomposer.deleteWorkspace && window.threatcomposer.deleteWorkspace(...)`.

Reply:
See #70